### DISCUSSION TITLE
Having trouble with GitKraken on pre-commit phase.

### DISCUSSION THREAD

**Discussion by emirxmertoglu** [coreybutler/nvm-windows#1061] [Support] [answered]
I installed nvm on a clean pc (no nodejs), and installed required node version with nvm. Everything works perfectly fine with Git Bash and Windows Terminal, like committing, pushing, pre-commit with husky and lint-staged etc. But there is an issue with GitKraken like that;
![error](https://github.com/coreybutler/nvm-windows/assets/22079417/d190e9d4-5aa2-4e70-bfed-db60f99e037a)

And here is my path;
![path](https://github.com/coreybutler/nvm-windows/assets/22079417/1defde2e-cb19-4e1d-a17a-53e74f1bf87a)
![cmd](https://github.com/coreybutler/nvm-windows/assets/22079417/d6c637b8-c614-4bc6-a3e1-3ed92dca719c)

Note that, I already tried this solution but it didn't work for me https://github.com/coreybutler/nvm-windows/discussions/572#discussioncomment-135399

**Comment by coreybutler** [comment, marked as answer]
This has nothing to do with NVM4W. You've already confirmed that Node is installed properly. NVM4W has nothing to do with the git command. GitKraken seems to be the problem. Notice the highlighted section of the error output:

<img width="650" alt="image" src="https://github.com/coreybutler/nvm-windows/assets/770982/8ab7ae1f-4bdb-4a05-8de5-7fca31fabca9">

An ASAR file is a compressed format (similar to zip) containing a bundle of files. The only tool I'm aware of that uses this extensively is Electron. It will extract files into folders just like what you see in your output. Apparently GitKraken isn't bundling yarn. Additionally, it is worth noting that Electron bundles its own Node binaries, so you're not using the Node versions installed on your system. I recommend reaching out to the GitKraken team for support.
